README.md

```diff
@@ -62,3 +60,6 @@ We will use docker swarm to install/run this image. Docker swarm generally consi
     ```
     docker stack deploy -c irods_jenkins.yml irods-jenkins
     ```
+
+## Miscellaneous
+If you're interested in modifying or enhancing this project, see [DEVELOPERS.md](/DEVELOPERS.md).
```

Comment:
i think the leading slash in the link is... extra.

Reply:
It is on purpose so that the browser understands that it is a relative path. Github understands it and does the correct thing when clicked.

I can try removing it and seeing what happens if you like?

Reply:
Removing the leading slash works as well. I'm not sure what is recommended for doing this.

Reply:
perhaps dot-slash?  `./DEVELOPERS.md`?

Reply:
sure.

Reply:
done.